File: dotnet/FeatureFlags/FeatureFlags/Models/Segment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FeatureFlags.Models
{
    public class Segment
    {
        public int Id { get; set; }

        public string Description { get; set; }

        public List<Constraint> Constraints { get; set; }

        public int Rank { get; set; }

        public int RolloutPercent { get; set; }
    }
}

File: java/src/main/java/com/excella/featureflagdemo/domain/Segment.java
package com.excella.featureflagdemo.domain;

import java.util.List;

public class Segment {

  private int id;
  private String description;
  private List<Constraint> constraints;
  private List<Distribution> distributions;
  private int rank;
  private int rolloutPercent;

  public int getId() {
    return id;
  }

  public void setId(int id) {
    this.id = id;
  }

  public String getDescription() {
    return description;
  }

  public void setDescription(String description) {
    this.description = description;
  }

  public List<Constraint> getConstraints() {
    return constraints;
  }

  public void setConstraints(List<Constraint> constraints) {
    this.constraints = constraints;
  }

  public List<Distribution> getDistributions() {
    return distributions;
  }

  public void setDistributions(List<Distribution> distributions) {
    this.distributions = distributions;
  }

  public int getRank() {
    return rank;
  }

  public void setRank(int rank) {
    this.rank = rank;
  }

  public int getRolloutPercent() {
    return rolloutPercent;
  }

  public void setRolloutPercent(int rolloutPercent) {
    this.rolloutPercent = rolloutPercent;
  }
}
